src/Entity/AlertBannerEntity.php

```diff
@@ -392,9 +392,12 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
    */
   public function getCacheContexts() {
 
+    $contexts = [];
+    $contexts = Cache::mergeContexts($contexts, ['url.path.is_front']);
```

Comment:
I don't think this line is necessary. Cache contexts are just strings so you can initialise it with `$contexts = ['url.path.is_front'];`